<ui version="4.0" >
 <class>Window</class>
 <widget class="QWidget" name="Window" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PyPI Information</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Search &amp;field:</string>
       </property>
       <property name="buddy" >
        <cstring>fieldCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="termsEdit" />
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="fieldCombo" >
       <item>
        <property name="text" >
         <string>name</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>version</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>author</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>author_email</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>maintainer</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>maintainer_email</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>home_page</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>license</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>summary</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>description</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>keywords</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>platform</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>download_url</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Search &amp;terms:</string>
       </property>
       <property name="buddy" >
        <cstring>termsEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Summary</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton" >
       <property name="text" >
        <string>&amp;Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldCombo</tabstop>
  <tabstop>termsEdit</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>308</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>325</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>termsEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>215</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
